stl/inc/xutility

```diff
@@ -1370,7 +1385,7 @@ public:
     static constexpr bool _Unwrap_when_unverified = _Do_unwrap_when_unverified_v<_BidIt>;
 
     template <class _Src, enable_if_t<_Wrapped_seekable_v<_BidIt, const _Src&>, int> = 0>
-    constexpr void _Seek_to(const reverse_iterator<_Src>& _It) {
+    constexpr void _Seek_to(const reverse_iterator<_Src>& _It) noexcept(noexcept(current._Seek_to(_It.base()))) {
         current._Seek_to(_It.current);

```

Comment:
Could this conditional `noexcept` match the function body exactly, by saying `_It.current`? I see no reason for it to differ here (unlike `operator[]`).